src/Compute/Compute.AlcWrapper/Compute.AlcWrapper.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.1.0" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.13.0" />
```

Comment:
Why is the build error showing that Az.Core is being downgraded from 1.25 to 1.24?

Reply:
resolved via https://github.com/Azure/azure-powershell/pull/19139